src/lib/util.js

```diff
@@ -0,0 +1,24 @@
+/*eslint no-undef: 0*/
+
+let browser;
+try {
+  if (chrome) {
+    browser = require('webextension-polyfill');
+  }
+} catch (e) {
+  if (e.name !== 'ReferenceError') {
```

Comment:
If you want to do it this way I believe `e instanceof ReferenceError` is more idiomatic.